doc/articles/Guides/CustomPlayer.md

```diff
@@ -0,0 +1,79 @@
+# Customizable Players
+
+Many games need character customization.  You may want to pick the color of the hair, eyes, skin, height, race, etc.
+
+By default Mirror will instantiate the player for you, which might prevent you from customizing it. However Mirror provides the option of overriding player creation and customize it.
```

Comment:
```suggestion
By default Mirror will instantiate the player for you. While that is convenient, it might prevent you from customizing it. Mirror provides the option of overriding player creation and customize it.
```